src/core/ext/filters/client_channel/lb_policy/xds/xds_cluster_manager.cc

```diff
@@ -505,6 +505,7 @@ void XdsClusterManagerLb::ClusterChild::DeactivateLocked() {
   // Set the child weight to 0 so that future picker won't contain this child.
   // Start a timer to delete the child.
   Ref(DEBUG_LOCATION, "ClusterChild+timer").release();
+  ExecCtx::Get()->InvalidateNow();
```

Comment:
This is not a repeated timer, so there can't be a loop here.

Reply:
removed